Models.py

```diff
@@ -90,6 +90,7 @@ def __init__(self, display_name, description, location, datetime, circle):
         circle.Scheduled.add(self)
         # Each member in the circle should be invited to the event.
         for member in circle.HasMember:
+            self.Invited.add(member)
```

Comment:
Current way of making events doesn't add this edge

Reply:
wdym? It looks like it worked in neo4j dev db?